js/nixos-site.js

```diff
@@ -1,3 +1,25 @@
+// FisherYates implementation
+$.fn.shuffleChildren = function(seed = Date.now()) {
+  var random = function(s) {
+    s = Math.sin(s) * 10000;
+    return s - Math.floor(s);
+  };
+
+  $.each(this.get(), function(index, el) {
+    var $el = $(el);
+    var $find = $el.children();
+    var len = $find.length;
+
+    while (--len) {
+      let randIndex = Math.floor(random(seed) * (len + 1));
```

Comment:
Here, the "random" source is called (re-initialised) with the seed at each
turn. That doesn't seem right.

Reply:
Ok I got your point. Will fix that.